### ISSUE TITLE
Gutenberg is loading slow, blank white page for over 300ms.

### ISSUE THREAD

**Issue by katsar0v** [WordPress/gutenberg#12673]
**Describe the bug**
I just updated to WordPress 5.0. Previously I had no issues and my website is quite faster in wp-admin and the front-end as well (response times under 50ms). After updating I went to view all posts and clicked on a post. I noticed the page loaded slower (whole page went blank white for fast 0.5s). Afterwards the new editor showed.

**To Reproduce**
1. Go to posts
2. Click on post
3. Check response time in network tools of you browser

**Expected behavior**
Obviously wordpress has been faster with the previous editor.

**Screenshots**
![image](https://user-images.githubusercontent.com/7236157/49640033-c404ca80-fa0c-11e8-82cc-7c06edf2102a.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 17
 - Browser Chrome Stable
 - Version [e.g. 22] 71.0.3578

**Comment by woorise** [comment]
I have notice the same issue.

**Comment by katsar0v** [comment]
`package-lock.json` is over 22k lines... I can only say "dafuq"

**Comment by brazenvoid** [comment]
Even when it was in beta, I have noticed ~10 second load times for existing posts. With 5.0.1, they have reduced to 5-7 seconds. But well, all our posts are 3000-6000 words long, the handling of which is a known performance issue.

The above are for complete page load. The editor takes about 0.6-1.1s to load.

**Comment by katsar0v** [comment]
@youknowriad Agree, thank you for the info. If I find something more specific, will open a special issue about it. Thanks again!
